lib/bots/notifybot.ts

```diff
@@ -85,7 +85,9 @@ interface TopicIssues {
 
 const NotifyBotPort = 8399; // Not a listed registered port
 
-const ConnectRE = /connects-to:[\s]+(#[0-9]+)/i;
+// https://help.github.com/articles/closing-issues-via-commit-messages/
+// https://help.waffle.io/faq/waffle-workflow/use-waffles-connect-keyword-to-connect-prs-to-issues
+const IssueReferenceRE = /(?:close[sd]?:?|fix(?:e[sd]{1})?:?|resolve[sd]?:?|connect(?:s|ed)?[\s]+(?:to)?)[\s]+(.*)/i;
```

Comment:
I think Github actually requires the '#' to reference this issues, so it needs to be tested for, and in fact NotifyBot differentiates HQ from local issues by the '#', so this RegExp needs to capture the '#' as well.
Length of the RegExp doesn't bother me. :)